.github/workflows/release-drafter.yml

```diff
@@ -0,0 +1,14 @@
+name: Release Drafter
+
+on:
+  push:
+    branches: [ main ]
```

Comment:
Please change this to main, master (we still use the master branch right now and haven't moved to main)